launch/app/pom.xml

```diff
@@ -48,14 +48,19 @@
       <type>pom</type>
       <scope>runtime</scope>
     </dependency>
-    
+    <dependency>
+      <groupId>org.openhab.addons.bom</groupId>
+      <artifactId>org.openhab.addons.bom.zigbee</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
```

Comment:
Just noticed another difference to the other bom dependencies:
```
<type>pom</type>
```
is missing here.

Wrt testing: Can't you (or @cdjackson)  test whether it is possible to use the Zigbee binding from within the IDE with that? After all, you want to use it so it is most important that it's working for you :-)

Reply:
In th zwave PR you mentioned that zwave is not a pom. So where is the difference?

https://github.com/openhab/openhab-distro/pull/933#discussion_r298796665

Reply:
The difference is that this is a pom. (in z-wave we are referencing the jar artifact, not a bom)

Reply:
If I add the `<type>pom</type>` eclipse does not complain anymore.

However, I cannot select the zigbee binding as a bundle to be started within `app.bndrun`.

Reply:
I had added the type again.